openlibrary/plugins/admin/code.py

```diff
@@ -829,6 +829,36 @@ def GET(self):
                 return f.read()
 
 
+class pd_dashboard:
+    def GET(self):
+        keys = web.ctx.site.things(
+            {
+                "type": "/type/object",
+                "key~": "/people/*/preferences",
+                "notifications": {"pda~": "*"},
```

Comment:
```suggestion
                "notifications": {"pda~": "*"},
                "limit": 100000,
```
Testing this query with a higher limit.